tjreports/site/controllers/reports.php

```diff
@@ -166,10 +166,24 @@ public function csvexport()
 
 		foreach ($data['colToshow'] as $eachColumn)
 		{
-			$calHeading    = strtoupper($eachColumn);
+			// Remove double Quotes from the data
+			$eachColumn       = str_replace('"', '', $eachColumn);
```

Comment:
why 3 different variables? eachColumn eachColumn2 final_eachColumn ?
only one should do.